gcloud/pubsub/client.py

```diff
@@ -50,6 +50,12 @@ def list_topics(self, page_size=None, page_token=None):
         See:
         https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/list
 
+        Example:
+
+        .. literalinclude:: pubsub_snippets.py
+           :start-after: [START client_list_topics]
+           :end-before: [END client_list_topics]
```

Comment:
Yeah I'd prefer just removing the snippets from the docstrings and relying on the RST docs.